src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -184,6 +184,8 @@ class InferenceServerClient:
         Object encapsulating various GRPC KeepAlive options. See
         the class definition for more information. Default is None.
 
+    creds: grpc.ChannelCredentials
+        A grpc.ChannelCredentials object to use for the connection.
```

Comment:
An optional grpc.ChannelCredentials object to use for the connection. The ssl, root_certificates, private_key and certificate_chain options will be ignored when using this option. Default is None.